### ISSUE TITLE
Remove convert?

### ISSUE THREAD

**Issue by madewild** [madewild/tac#81]
Issues with pdf2text

**Comment by madewild** [comment]
Get rid of pdf2text and keep pypdf2 only

**Comment by madewild** [comment]
works with textract?

**Comment by DeLeb86** [comment]
@madewild it looks like this one is already covered right ?

**Comment by madewild** [comment]
Indeed, but should we get rid of textract altogether in https://github.com/madewild/tac/blob/main/module6/s1_extraction.ipynb ?

**Comment by DeLeb86** [comment]
@madewild  : Textract can be replaced by several libraries : Markitdown[docx,pdf,pptx,xls,xlsx], docx2txt, striprtf, odfpy, easyocr, openai-whisper
they are all full python libraries with no need to install binaries or dependencies.
It's a bit more tedious than just use textract but it works to show text extraction from different file formats.

**Comment by madewild** [comment]
Great for me, just test to see which one works with most formats with the least depedencies

**Comment by DeLeb86** [comment]
We first need to merge the PR regarding the migration to UV (https://github.com/madewild/tac/pull/117). 
To correctly clean the dependencies

**Comment by madewild** [comment]
Sorry missed that, reviewed and merged
